pkg/kubectl-explain/components/SlideInPanel.vue

```diff
@@ -0,0 +1,434 @@
+<script>
+import ExplainPanel from './ExplainPanel';
+import { KEY } from '@shell/utils/platform';
+import { expandOpenAPIDefinition, getOpenAPISchemaName, makeOpenAPIBreadcrumb } from '../open-api-utils.ts';
+
+const HEADER_HEIGHT = 55;
+
+export default {
+  components: { ExplainPanel },
+
+  props: {
+    schema: {
+      type:    Object,
+      default: () => {}
+    },
+    $t: {
+      type:     Function,
+      required: true,
+    }
+  },
+
+  data() {
+    return {
+      isOpen:         false,
+      definition:     undefined,
+      busy:           true,
+      error:          false,
+      expandAll:      false,
+      isResizing:     false,
+      resizeLeft:     '',
+      resizePosition: 'absolute',
+      width:          '33%',
+      right:          '-33%',
+      breadcrumbs:    undefined,
+      definitions:    {},
+      noResource:     false,
+      notFound:       false,
+    };
+  },
+
+  computed: {
+    top() {
```

Comment:
There's a slightly competing method of discovering the height of the banners in https://github.com/rancher/dashboard/pull/9768/files. That one looks directly at the setting and creates a pixel height given 2em and font size.

Personal style would be the other way, but it's brittle given changes to styling (due to implementation, there'd way to fix that) and having to fetch resources.

Once both are in, that other method should change to match this method